src/main/java/baritone/utils/schematic/format/DefaultSchematicFormats.java

```diff
@@ -65,6 +66,26 @@ public IStaticSchematic parse(InputStream input) throws IOException {
                     throw new UnsupportedOperationException("Unsupported Version of a Sponge Schematic");
             }
         }
+    },
+
+    /**
+     * The Litematica schematic specification. Commonly denoted by the ".litematic" file extension.
+     */
+    Litematica("litematic") {
+        @Override
+        public IStaticSchematic parse(InputStream input) throws IOException {
+            NBTTagCompound nbt = CompressedStreamTools.readCompressed(input);
+            int version = nbt.getInteger("Version");
+            switch (version) {
+                case 4:
+                    return new LitematicaSchematic(nbt);
+                case 5:
+                case 6:
+                    throw new UnsupportedOperationException("This Schematic Verion is to new.");
```

Comment:
too

Reply:
what do you mean here? there are 3 schematic version used atm v4 for 1.12 v5 for 1.13-1.17 and v6 1.18+.
it is intentional that bouth case 5&6 point to the unsupportet version exception for beeing too new